apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -151,7 +151,7 @@ class StageLockDialog extends React.Component {
     const responsiveHeight = {
       maxHeight: window.innerHeight * 0.8 - 100
     };
-    const hasSelectedSection = this.props.selectedSectionId !== '';
+    const hasSelectedSection = this.props.selectedSectionId !== null;
```

Comment:
NIT: i think `!!this.props.selectedSectionId` would be the more "javascript-y" way to do this.